notes/2020-04-02.md

```diff
@@ -0,0 +1,148 @@
+
+### Introduction of attendees (5m, Lee)
+
+
+### Agree to Membership Agreement, Participation Guidelines and Code of Conduct (1m, Lee)
+
+*   Specification Membership Agreement
+*   Participation Guidelines
+*   Code of Conduct
+
+### Determine volunteers for note taking (2m, Lee)
+
+*   Benjie
+
+### Review agenda (2m, Lee)
+
+*   No changes.
+
+### GraphQL Foundation Turns 1! See the Annual Report: foundation.graphql.org/annual-report-2019
+
+*    Looks like Lee wrote it, but really it was Brian and Chris who did most of the work
+*    Check it out!
+*    Share it amongst your communities
+*   Lee: Members can join the GraphQL Foundation directly without needing to be a member of the Linux Foundation
+*   Robert: Amazon would like to include a lot more information in future beyond the Amplify stack - want to make it clear that Amazon uses GraphQL in a lot more places than just Amplify
+
+### Custom scalar specification URIs (5m, Evan Huus / Matt Farmer)
+
+*   Evan: Status check, still waiting on graphql-js (though that's started moving forward again now)
+*   Ivan: Released 15.0.0, in a week would like to add a new feature release. Will try and finish review today.
+*   Ivan: Question: if we change the "specified by" URL on a scalar, technically this is a breaking change because it's an identifier on the client side, and the client might use it. Should we treat this as a breaking change?
+*   Ivan: if URL changes, because it's opaque, we cannot detect if it's breaking or not
+*   Morris: we don't know what it means when the string changes - it may or may not be a breaking change.
+*   Lee: we also need to clarify what we mean by "breaking". Really there's different categories of breaking changes: if you take an identical query to a server after a change - will you get something meaningfully different, or will the query no longer be valid? This is a truly breaking change - breaks backwards compatibility. (This would be a high value thing to define as a breaking change.) There's also "dangerous change" which is not "breaking". Client side tooling relies on introspection to do stuff, if we change these things it may cause headaches for client-side tooling. Code-generation at Facebook inspired the initial "dangerous change" rules. Think of these as "forward breaking changes" as clients will pick these up in the future and may have issues.
+*   Ivan: I agree with Lee - usual clients don't use introspection in production, so once the app is compiled and released most clients wouldn't notice the difference.
+*   Robert: can we try and document the differences between these different kinds of breaking changes?
+*   Lee: absolutely, that's definitely a gap in the spec.  
+*   Matt M: even some of the "breaking changes" are not necessarily always breaking changes - e.g. changing a type to an interface may not break client queries (but it also might, because __typename may change)  
```

Comment:
This broke the highlighting of the raw markdown.

```suggestion
*   Matt M: even some of the "breaking changes" are not necessarily always breaking changes - e.g. changing a type to an interface may not break client queries (but it also might, because `__typename` may change)  
```

Reply:
Good eyes, thanks @benjie!